File: SoftUni-Python-Fundamentals/p02_data_types_and_variables/exercises/gladiator_expenses.py
lost_fights = int(input())
helmet_price = float(input())
sword_price = float(input())
shield_price = float(input())
armor_price = float(input())
expenses = 0
shields_broken = 0
for fight in range(1, lost_fights+1):
    if fight % 2 == 0:
        expenses += helmet_price
    if fight % 3 == 0:
        expenses += sword_price
    if fight % 2 == 0 and fight % 3 == 0:
        expenses += shield_price
        shields_broken += 1
    if shields_broken % 2 == 0 and shields_broken != 0:
        expenses += armor_price
        shields_broken = 0
print(f"Gladiator expenses: {expenses:.2f} aureus")

File: SoftUni-JS-Front-End/jsSyntaxFundamentalsMoreExercises/gladiatorExpenses.js
function gladiatorExpenses(lostFights, helmetPrice, swordPrice, shieldPrice, armorPrice) {
    let expenses = 0
    expenses += Math.floor(lostFights / 2) * helmetPrice
    expenses += Math.floor(lostFights / 3) * swordPrice
    expenses += (Math.floor(lostFights / 6)) * shieldPrice
    expenses += (Math.floor(lostFights / 12)) * armorPrice
    console.log(`Gladiator expenses: ${expenses.toFixed(2)} aureus`)
}

gladiatorExpenses(23, 12.5, 21.5, 40, 200)